src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, number) {
+      callback.call(this, number);
+
+      return this;
```

Comment:
This is the most unusual implementation of "advanced calculator" I've seen 👍 
We've definately had no information about `call` on the lectures.